src/focus.service.ts

```diff
@@ -508,7 +508,7 @@ export class FocusService {
    */
   private isFocusable(el: Element) {
     const role = el.getAttribute('role');
-    const tabIndex = el.getAttribute('tabIndex');
+    const tabIndex = el.getAttribute('tabindex');
```

Comment:
It returns Element because it itself is an Element. It's OK as-is I guess, I want to go through and change the types later though